dev.html

```diff
@@ -2799,6 +2952,296 @@ <h2>Conditions</h2>
+
+              <#student>
+                a rr:TriplesMap ;
+                rml:subjectMap [
+                  rr:template "http://example.org/student/{Student}";
+                ] ;
+                rr:predicateObjectMap [
+                  rr:predicate rdf:type ;
+                  rml:object ex:Student
+                ] .
+            </pre>
+        </div>
+      </div>
+
+      Quoted triples can be non asserted in the RDF-star graph using the <code>QuotedNonAsserted</code> key
+
+      <div id="tabs-62" class="nanotabs">
+        <ul>
+          <li><a href="#tab62-1" class="selected">YARRRML</a></li>
+          <li><a href="#tab62-2">RML</a></li>
+        </ul>
+        <div id="tab62-1">
+          <pre id="example-fn-oneline-no-prefix" class="example" title="Quoted non-asserted mapping in object">
+            mappings:
+              person:
+                subject: http://example.org/person/$(Name)
+                predicateobjects:
+                 - predicates: ex:confirms
+                   objects:
+                    - quotedNonAsserted: student
+              student:
+                subject: http://example.org/student/$(Student)
+                predicateobjects:
+                 - [a, ex:Student]
+          </pre>
+        </div>
+        <div id="tab62-2">
+            <pre id="example-fn-oneline-rml-no-prefix" class="example" title="Quoted non-asserted mapping in object">
+              <#person>
+                a rr:TriplesMap ;
+                rml:subjectMap [
+                  rr:template "http://example.com/person/{Name}"
+                ] ;
+                rr:predicateObjectMap [
+                  rr:predicate foaf:firstName ;
+                  rml:objectMap [
+                    rml:quotedTriplesMap <#student>;
+                  ];
+                ] .
+
+              <#student>
+                a rml:NonAssertedTriplesMap, rr:TriplesMap ;
+                rml:subjectMap [
+                  rr:template "http://example.org/student/{Student}";
+                ] ;
+                rr:predicateObjectMap [
+                  rr:predicate rdf:type ;
+                  rml:object ex:Student
+                ] .
+            </pre>
+        </div>
+      </div>
+
+      Joins can also be defined in the same way with <code>quoted</code> and <code>quotedNonAsserted</code>
+
+      <div id="tabs-43" class="nanotabs">
+        <ul>
+          <li><a href="#tab63-1" class="selected">YARRRML</a></li>
+          <li><a href="#tab63-2">RML</a></li>
+        </ul>
+        <div id="tab63-1">
+          <pre id="example-fn-oneline-no-prefix" class="example" title="Join in quoted mapping in subject">
+            mappings:
+              person:
+                subject:
+                 - quoted: student
+                   condition:
+                   function: equal
+                   parameters:
+                     - [str1, $(condition1)]
+                     - [str2, $(condition2)]
```

Comment:
Ah I see now why the syntax is not a function like with joins.

Reply:
this might be a typo, the syntax is the same, similar functions are allow as conditions

Reply:
solved in my fork